controllers/PostController.js

```diff
@@ -106,6 +106,26 @@ const unlikePost = async (req, res) => {
   }
 };
 
+const updatePost = async (req, res) => {
```

Comment:
Do you think `editPost` is a better name for what's going on? Not much of a difference but maybe it's more specific? @NasihNazeem @christopherkinyua what do you guys think? I don't care too much either way but if y'all agree we can change the name of this controller/query/endpoint

Reply:
I can make it `editPost` as the ticket is for edit posts.

Reply:
Done